dmithamo/mydiary
# #159100964 Build out the 'mydiary/api/v1/entries/' Endpoint

#### What does this PR do?
This PR builds out the endpoint defined in the title above, so that the `API` can now respond
to requests at `GET mydiary/api/v1/entries/` .
#### To test this :
1. Clone the repo, and navigate top the root directory of the API, i.e., enter `cd API` in Terminal.
(It may be necessary to change to the appropriate git branch : `git checkout ft-api-fetch-all-entries-159100964` )
2. Install the API's dependencies by entering `pip install -r requirements.txt` in Terminal
3. Set the API's entry point by entering `export FLASK_APP=app/__init__.py` in Terminal.
4. Start the server by entering `flask run` in Terminal

This endpoint can then be tested. An example of the expected response is :
```
{
    "message": "No entries yet."
}
```
`Response status code = 200 OK`

#### Screenshots 
![api-response-1](https://user-images.githubusercontent.com/33679184/42866640-c77b6482-8a75-11e8-9cf0-668bed1da30b.png)
